Fix for grolbp segfaulting when given unexpected arguments (CR #18122012).

This fix is already integrated into the groff git repository upstream.

--- groff-1.19.2/src/devices/grolbp/lbp.cpp.orig	2014-01-23 10:59:06.859623493 -0800
+++ groff-1.19.2/src/devices/grolbp/lbp.cpp	2014-01-23 10:59:47.816730501 -0800
@@ -719,6 +719,7 @@
     do_file("-");
   while (optind < argc)
     do_file(argv[optind++]);
-  lbpputs("\033c\033<");
+  if (lbpoutput)
+    lbpputs("\033c\033<");
   return 0;
 }
